unity/ARUI/Assets/Scripts/ARUI/UIComponents/ConfirmationDialogue.cs

```diff
@@ -5,14 +5,22 @@
 using TMPro;
 using UnityEngine;
 using UnityEngine.Events;
+using RosMessageTypes.Angel;
+
+
+[System.Serializable]
+public class InterpretedAudioUserIntentEvent : UnityEvent<InterpretedAudioUserIntentMsg>
```

Comment:
Could use a documenting comment for the purpose of the class.

Reply:
Yes, I am currently working on this class's documentation in my fork.

Reply:
I appreciate it! This instance is more for @joshanderson-kw with this new class that was introduced. I think it's likely just a simple sentence to add.